src/Zend/Xml/Security.php

```diff
@@ -65,11 +65,10 @@ public static function loadXmlErrorHandler($errno, $errstr, $errfile, $errline)
      * Scan XML string for potential XXE and XEE attacks
      *
      * @param   string $xml
-     * @param   DOMDocument $dom
      * @throws  Zend_Xml_Exception
      * @return  SimpleXMLElement|DOMDocument|false
      */
-    public static function scan($xml, DOMDocument $dom = null)
+    public static function scan($xml, DOMDocument|null $dom = null)
```

Comment:
What made you prefer this syntax over `?DOMDocument` ?

Reply:
The nullable type syntax (`?DOMDocument`) was added in PHP 8.1 and these packages still support 8.0. I would've dropped 8.0 if necessary, but this syntax is effectively the same, so figured I'd use it and keep 8.0 support for the time being (even though it's EOL).

Reply:
I forgot it was added only later on!

Reply:
This syntax was added in PHP 7.1, not 8.1 !